src/components/viewToolbar/viewToolbarSelect.js

```diff
@@ -106,6 +106,7 @@ const ViewToolbarSelect = ({
       position={position}
       selectedOptions={selectedOption}
       data-test={`toolbarSelect_${filter}`}
+      ouiaId={`toolbarSelect_${filter}`}
```

Comment:
question: unclear why both of these `data-test` and `ouiaId` are being used, are both necessary?

Reply:
This is mostly for consistency on Camayoc side. Specifying `ouiaId` on pretty much all interactive elements we care about allows us to use it consistently in automation, and probably introduce a number of helpers and heuristics (i.e. "if not specified otherwise, assume string is ouia-id"). When some elements have `data-test` and some have `ouia-id`, it only increases effort needed to maintain automation on Camayoc side.